src/ddmd/visitor.d

```diff
@@ -84,6 +84,11 @@ extern (C++) class Visitor
         visit(cast(Statement)s);
     }
 
+    void visit(ForwardingScopeStatement s)
```

Comment:
I did some work on this, but I have no way to be confident that everything matches. Will the autotester catch mismatches?